backend/.env.example

```
@@ -62,3 +62,15 @@ STRIPE_WEBHOOK_SECRET=whsec_xxxx
+
+# Server-side Google Geocoding API key. Do not expose this in NEXT_PUBLIC_* envs.
+# Restrict in Google Cloud to Geocoding API only.
+GEOCODING_API_KEY=XXXXXX
+
+# Relationship map — admin-only home marker (optional).
+# Returned by the admin-gated `relationshipHomeLocation` GraphQL query.
+# DO NOT expose any of these via NEXT_PUBLIC_* — they would leak into the browser bundle.
+# Leave blank to disable the home marker on /admin/relationship.
+RELATIONSHIP_HOME_LAT=
+RELATIONSHIP_HOME_LNG=
+RELATIONSHIP_HOME_LABEL=Home base
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The RELATIONSHIP_HOME_LABEL key should go before the RELATIONSHIP_HOME_LAT key

(UnorderedKey)